src/visualizer.jl

```diff
@@ -108,6 +108,9 @@ function update_tree!(core::CoreVisualizer, cmd::SetAnimation, data)
     core.tree.animation = data
 end
 
+function update_tree!(core::CoreVisualizer, cmd::TakeScreenshot, data)
+    core.tree.animation = data
```

Comment:
This doesn't look right. The tree update just ensures that any commands used to create the scene will be replayed when a new browser window connects or reloads. But we don't want to re-trigger taking a screenshot, so this particular method shouldn't do anything. I believe it should be sufficient to keep this method but leave the body of it blank (or explicitly put a body of `nothing` to make it more clear that nothing needs to happen).